testsuite/forge/src/runner.rs

```diff
@@ -162,6 +162,16 @@ impl<'cfg> Default for ForgeConfig<'cfg> {
             initial_version: InitialVersion::Newest,
         }
     }
+
+    pub fn cloud_backend() -> Self {
+        Self {
+            public_usage_tests: &[],
+            admin_tests: &[],
+            network_tests: &[],
+            initial_validator_count: NonZeroUsize::new(30).unwrap(),
+            initial_version: InitialVersion::Newest,
+        }
+    }
```

Comment:
Lets not make these changes. We should keep the `Default` impl as is and we shouldn't have a `cloud_backend` default in this create, instead create this as a function and stick it in the forge-cli crate.

Reply:
Please revert the changes to this file. this can be achieved with

```
ForgeConfig::default().with_initial_validator_count(30)
```